packages/parser/test/specs/invalid/no-paths-or-webhooks.yaml

```diff
@@ -1,4 +1,4 @@
-openapi: '3.1'
+openapi: '3.1.0'
```

Comment:
Changed this because this spec is testing validation for when `paths` and `webhooks` aren't present but `openapi` not being `3.1.0` was throwing spec validation.